datahub-web-react/src/app/ingest/source/ExecutionRequestDetailsModal.tsx

```diff
@@ -1,48 +1,147 @@
-import { message, Modal, Typography } from 'antd';
-import React from 'react';
+import { DownloadOutlined } from '@ant-design/icons';
+import { Button, message, Modal, Typography } from 'antd';
+import React, { useEffect, useState } from 'react';
 import styled from 'styled-components';
 import { useGetIngestionExecutionRequestQuery } from '../../../graphql/ingestion.generated';
+import { ANTD_GRAY } from '../../entity/shared/constants';
+import { downloadFile } from '../../search/utils/csvUtils';
 import { Message } from '../../shared/Message';
+import IngestedAssets from './IngestedAssets';
+import {
+    getExecutionRequestStatusDisplayColor,
+    getExecutionRequestStatusDisplayText,
+    getExecutionRequestStatusIcon,
+} from './utils';
 
 const Section = styled.div`
     display: flex;
     flex-direction: column;
     padding-bottom: 12px;
 `;
 
-const SectionHeader = styled(Typography.Text)`
+const SectionHeader = styled(Typography.Title)`
     &&&& {
         padding: 0px;
         margin: 0px;
         margin-bottom: 12px;
     }
 `;
 
+const HeaderSection = styled.div``;
+
+const StatusSection = styled.div`
+    border-bottom: 1px solid ${ANTD_GRAY[4]};
+    padding: 16px;
+    padding-left: 30px;
+    padding-right: 30px;
+`;
+
+const IngestedAssetsSection = styled.div`
+    border-bottom: 1px solid ${ANTD_GRAY[4]};
+    padding: 16px;
+    padding-left: 30px;
+    padding-right: 30px;
+`;
+
+const LogsSection = styled.div`
+    padding-top: 16px;
+    padding-left: 30px;
+    padding-right: 30px;
+`;
+
 type Props = {
     urn: string;
     visible: boolean;
     onClose: () => void;
 };
 
 export const ExecutionDetailsModal = ({ urn, visible, onClose }: Props) => {
+    const [showExpandedLogs, setShowExpandedLogs] = useState(false);
     const { data, loading, error } = useGetIngestionExecutionRequestQuery({ variables: { urn } });
     const output = data?.executionRequest?.result?.report || 'No output found.';
 
+    useEffect(() => {
+        if (output.length > 100) {
+            setShowExpandedLogs(false);
+        }
+    }, [output, setShowExpandedLogs]);
```

Comment:
also small -> do you want to set the default state to `true` so that we then set it to `false` right away with this useEffect if the logs length is large enough? otherwise the only way this would be true is to click "Show More" even if the logs are only a couple of words at first.

Reply:
Oh yeah good idea!